modules/RTC/BridgeChannel.js

```diff
@@ -276,11 +277,17 @@ export default class BridgeChannel {
      * Set events on the given RTCDataChannel or WebSocket instance.
      */
     _handleChannel(channel) {
+        // This handler gets called right after the channel is created, so we can mark the
+        // start here.
+        perfMetrics.markStart(perfMetrics.METRICS.BRIDGE_CHANNEL_OPEN);
+
         const emitter = this._eventEmitter;
 
         channel.onopen = () => {
             logger.info(`${this._mode} channel opened`);
```

Comment:
Maybe now this log is redundant?

Reply:
It's still good to know, regardless of the measurement, IMHO.